### PR TITLE
Form Complete Metadata Column for Supertibble

### LINKED ISSUE #183: [FEATURE]: `form_complete_pct` in supertibble
# Feature Request Description

Add an additional metadata column to the supertibble, `form_complete_pct`, showing the percentage of `form_status_complete` for each data tibble.

### Checklist

- [x] The issue is atomic
- [x] The issue description is documented
- [x] The issue title describes the problem succinctly
- [x] Developers are assigned to the issue
- [x] Labels are assigned to the issue

### PR DESCRIPTION
**Pull request by rsh52**
# Description
This is a very light PR adding in `form_complete_pct` as a metadata column to the supertibble. It summarizes the average of "Complete"s/2's for a given form (it does not take into account unverifieds, etc. which I don't think we can capture in one column).

# Proposed Changes 
List changes below in bullet format:

- Add `form_complete_pct` to supertibble metadata columns
- Update `write_redcap_xlsx` to appropriately handle percentage output (similar to `data_na_pct`)
- Update README image to show new column.

### Issue Addressed
**Closes #183**

### PR Checklist
Before submitting this PR, please check and verify below that the submission meets the below criteria:

- [x] New/revised functions have associated tests
- [NA] New/revised functions that update downstream outputs have associated static testing files (`.RDS`) updated under `inst/testdata/create_test_data.R`
- [NA] New/revised functions use appropriate naming conventions
- [x] New/revised functions don't repeat code
- [x] Code changes are less than **250** lines total
- [x] Issues linked to the PR using [GitHub's list of keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] The appropriate reviewer is assigned to the PR
- [x] The appropriate developers are assigned to the PR
- [x] Pre-release package version incremented using `usethis::use_version()`

# Code Review
This section to be used by the reviewer and developers during Code Review after PR submission

### Code Review Checklist

- [ ] I checked that new files follow naming conventions and are in the right place
- [ ] I checked that documentation is complete, clear, and without typos
- [ ] I added/edited comments to explain "why" not "how"
- [ ] I checked that all new variable and function names follow naming conventions
- [ ] I checked that new tests have been written for key business logic and/or bugs that this PR fixes
- [ ] I checked that new tests address important edge cases

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: REDCapTidieR
 Type: Package
 Title: Extract 'REDCap' Databases into Tidy 'Tibble's
-Version: 1.1.0
+Version: 1.1.0.9000
```
Comment (ezraporter) [inline DESCRIPTION:4]: I think we actually don't want to bump this if we plan to release this as part of `1.1.0`

**Review by ezraporter — APPROVED** [review]
🚀

File: DESCRIPTION (hunk 1 above)
Comment (rsh52) [reply DESCRIPTION:4]: Oh, yea right. 👍
